src/components/about/Beliefs.tsx

```diff
@@ -0,0 +1,169 @@
+'use client'
+import React, { Fragment, useState } from 'react'
+import { PageTitle } from '@/components/PageTitle'
+import { Container } from '@/components/Container'
+import { Listbox, Transition } from '@headlessui/react'
+import { CheckIcon, ChevronUpDownIcon } from '@heroicons/react/20/solid'
+import clsx from 'clsx'
+import { pad } from '@/utils/helpers'
+
+interface BeliefItem {
+  id: string
+  name: string
+  verse: string
+  description: string
+}
+const Beliefs = () => {
+  const beliefs = Array.from(Array(10).keys()).map((i) => ({
+    id: pad(String(i + 1)),
+    name: 'Name of belief',
+    verse: 'Supporting verse',
+    description:
+      'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Dicta iure beatae debitis, nobis repudiandae doloribus sit voluptatibus facilis voluptatum, eos corrupti quaerat, sunt perspiciatis enim expedita! Iure aspernatur alias maxime?',
+  }))
+
+  const [active, setActive] = useState(beliefs[0] ?? null)
+
+  const onOptionClick = (option) => {
+    setActive(option)
+  }
+
+  return (
+    <section className="bg-surface-dark py-8">
+      <PageTitle title={'Our Beliefs'} />
+      <Container>
+        <div className="flex w-full md:hidden my-8">
+          <Select
+            options={beliefs}
+            selected={active}
+            onChange={onOptionClick}
+          />
+        </div>
+        <div className="grid grid-cols-1 gap-8 lg:grid-cols-9">
+          <div className="md:p-8 lg:col-span-4">
+            <div className="h-full rounded-md bg-white p-8">
+              <div className="my-4 text-4xl font-semibold text-muted">
+                {active?.id}
+              </div>
+              <div className="text-2xl font-semibold">{active?.name}</div>
+              <div className="mb-4 text-lg text-muted">
+                &ldquo;{active?.verse}&rdquo;
+              </div>
+              <p>{active?.description}</p>
+            </div>
+          </div>
+
+          <div className="hidden md:block lg:col-span-5">
+            <div className=" rounded md:p-8">
+              <div className="grid grid-cols-1 gap-4  sm:grid-cols-2">
+                {beliefs.map((item) => {
+                  return (
+                    <Belief
+                      onClick={onOptionClick}
+                      key={item.name}
+                      item={item}
+                    />
+                  )
+                })}
+              </div>
+            </div>
+          </div>
+        </div>
+      </Container>
+    </section>
+  )
+}
+
+export default Beliefs
+
+interface BeliefProps {
+  item: BeliefItem
+  onClick: Function
+}
+
+const Belief = ({ item, onClick }: BeliefProps) => {
```

Comment:
the page component should always come at the top of the file, and any other supporting components below it.  Also let's stick with one option, either function components or const components.

Reply:
the beliefs component is the main one , and it is on top